### DISCUSSION TITLE
Part Cooling Fan Mod

### DISCUSSION THREAD

**Discussion by NuclearPhoenixx** [1coderookie/KobraGoNeoInsights#15] [Q&A]
Hi there, I was wondering what type of part cooling shroud you guys use instead of the default one that ships with the Neo.

I printed myself this one, because it mentiones several improvements on the other ones, but I've not noticed a big difference to be entirely honest: https://www.printables.com/de/model/470004-anycubic-kobra-neo-fan-duct
I'm printing only in PLA at the moment, so maybe it's more important for other filament types.

**Comment by 1coderookie** [comment]
I'm using (a slightly modified version to fit my longer heatbreak setup of) this one: https://www.printables.com/de/model/498457-anycubic-kobra-neo-fan-duct  
Will try this one soon as well tho as the guy keeps working on it: https://www.thingiverse.com/thing:5985654

**Reply by 1coderookie** [reply]
Forgot: especially for printing PLA a proper and sufficient cooling is important as that needs to be cooled down quickly.  
Sure, PETG needs a good cooling as well when printing e.g. overhangs, but if not printing that, you can still print 'fine' as it needs less (or no cooling at all) compared to PLA.

**Reply by NuclearPhoenixx** [reply]
> I'm using (a slightly modified version to fit my longer heatbreak setup of) this one: https://www.printables.com/de/model/498457-anycubic-kobra-neo-fan-duct

How did you print that one? The overhangs in layer 31 don't work for me at all. Or did you modify that part as well?

**Reply by 1coderookie** [reply]
I had kind of a hard time with those as well, especially as I was using PETG. But in the end it worked out quite well. As you already use an 'improved' fan duct now, printing this one might become possible as well now.  
Or just try the other circular one I linked to - as I said, I'll try that one next as well, as the guy is currently working on it and keeps improving it.

**Reply by NuclearPhoenixx** [reply]
Hmm... Just tried it with zigzags but unfortunately the filament just sagged through the gaps :/
Might have to try something else.

**Reply by 1coderookie** [reply]
Maybe you need to dial in the bridging abilities/settings better in the first place. Find a specific model for that and try to tune your slicer settings, like the amount of flow, print speed, fan speed and so on specifically when bridging occurs.  
I'm kinda lazy when it comes down to finetune slicer settings, but iirc I had to reduce the flowrate for example. 

And it's definitely easier to do with PLA, which is sufficient for the fan duct as well - as long as you don't start printing e.g. PETG with 80° bedtemps, cuz that will soften the PLA then and make the fan duct sag.  
I printed one from PLA first, worked fine when using PLA, but when I started to print PETG I had to print one from PETG as well. Which then was easier though as the new fan duct improved the cooling abilities..

**Reply by NuclearPhoenixx** [reply]
What slicer settings were you using to print the fan?

**Reply by 1coderookie** [reply]
Puuuh, I really don't remember, that's quite long ago and I was experimenting with different settings. I only remember that I had to reduce the flowrate for bridging and used 100% fan for it. Besides that, I'm using a 0.6 nozzle, so that's different to stock 0.4 anyway. Not to mention that I'm quite lazy when it comes down to finetune settings to perfection.. :( ;)

**Reply by NuclearPhoenixx** [reply]
Hm, it's really not working for me. It's always failing at the overhang, although I don't have problems printing overhangs in general. It's just that it tries to attach the filament in mid air to the wall. If there was just a single track under the wall where it tries to attach to, it would certainly work. Maybe I'll adjust the model to fix that for me. No idea how it worked for you, though. The degree of success changes a lot by what top bottom layer pattern I use. Or maybe I'll try the updated version instead: https://www.printables.com/de/model/540618-anycubic-kobra-neo-fan-duct-updated

**Comment by NuclearPhoenixx** [comment]
I know now why I've got problems with the specific fan. The layer it tries to bridge over in the inner circle has nothing under it to support it. It tries to attach the bridging to the wall, doesn't stick and fails. The longer lines that run between the outer walls bridge just fine because there is something under them to attach to at the ends.

![grafik](https://github.com/1coderookie/KobraGoNeoInsights/assets/17517687/8da004eb-6cf2-443f-8206-16254bc13c0a)

**Reply by NuclearPhoenixx** [reply]
I actually managed to fixed it. Problem was the extra skin wall count was set to one (blue circular line in the image) together with only 5% skin overlap percentage. Disabled extra wall and increased skin overlap a bit and not it manages to attach to the wall. I'll report my experience with this new fan.

**Reply by 1coderookie** [reply]
Thanks, might be good and helpful for other users if you'd post that as a comment at the model's page as well :)

**Reply by NuclearPhoenixx** [reply]
> I'll report my experience with this new fan.

The fan is pretty weird tbh, it almost scrapes the bottom v-slot wheel of the printhead and the nozzle isn't exactly in the center of the whole construction anymore. Also the plastic printhead case doesn't fit anymore at all. Other than that, I can feel the airflow is pretty good.

![20231114_234019](https://github.com/1coderookie/KobraGoNeoInsights/assets/17517687/7619ca76-b784-4d52-9220-e2d898fa5c1a)
![20231114_234004](https://github.com/1coderookie/KobraGoNeoInsights/assets/17517687/808ba33a-1936-44d0-99d0-32590290093f)

I'm gonna try some more fans and report if I find something new or something that maybe fits a bit better.

**Reply by 1coderookie** [reply]
Correct, you can't use the blue plastic housing of the head anymore with these.  
I didn't had issues with the wheels, but I guess it's cuz I edited the model a bit to make it reach down further. First of all cuz I'm using a slightly longer heatbreak than the stock one and second cuz imho the duct is a bit too high anyway when looking at your last picture.

**Reply by NuclearPhoenixx** [reply]
Yeah true.

**Reply by NuclearPhoenixx** [reply]
>  I didn't had issues with the wheels, but I guess it's cuz I edited the model a bit to make it reach down further. First of all cuz I'm using a slightly longer heatbreak than the stock one and second cuz imho the duct is a bit too high anyway when looking at your last picture.

Do you have an STL for that? ;)

**Reply by 1coderookie** [reply]
I *should* have it *somewhere*, but I actually didn't find in in my folders yet :(   
But you should be able to just elongate those two mounting holes with a drillbit for being able to slide it down further a bit.

**Reply by NuclearPhoenixx** [reply]
> But you should be able to just elongate those two mounting holes with a drillbit for being able to slide it down further a bit.

Oh yeah good point, didn't think of that :D

**Comment by NuclearPhoenixx** [comment]
I'm going to compare these three upgraded fan ducts today and I'll report on the results of the same fan test print for all three.

1. https://www.printables.com/de/model/498457-anycubic-kobra-neo-fan-duct
2. https://www.thingiverse.com/thing:5985654
3. https://www.printables.com/de/model/470004-anycubic-kobra-neo-fan-duct

So far the Kobra NEO fan duct upgrade V9.3 (number 2) looks to restrict air flow too much. Also, the lower right screw for the fan is at the wrong position and the duct looks a bit too high all-in-all. The spacing around the nozzle is a lot better than with the next model tho. I've had fairly good results with the "Anycubic Kobra Neo fan duct" (number 1) by [Marius Avasiloaie](https://www.printables.com/de/@MariusAvasiloaie), but it's pretty large in circumference which means it's pretty far away from the nozzle honestly. It's also not centered around the nozzle like it should be I guess. Also, like I said in a previous post, it can touch the V slot wheel of the printhead a bit if not slightly adjusted when mounting. It has good airflow tho.
I've been using the third model "Anycubic Kobra Neo Fan Duct" (number 3) by [Kaan Okcu](https://www.printables.com/de/@KaanOkcu_682657) for most of the time now and while it's good in size and airflow, it's only 180°, meaning you'll get blind spots. However, it's got the best mount to the fan of all the three IMO.

Unfortunately, I think all three of them are a tad too high above the nozzle. I'd definitely give em like an additional 2 millimeters down or so. For testing I'm using the following cooling fan torture test with a central fine point needle structure and four overhangs to each side with up to 80°: https://www.printables.com/de/model/180150-cooling-fan-test-torture-test

Good news first, all three of them completed the print without major failures. The overhangs look a lot better than the stock Ender 3 ones that are referenced in the printables description. Although I'm sure there is a lot of play within all the other variables (e.g. filament, splicer settings, etc.). It's hard to capture the results in photos, because they're all so close to each other and my camera is slightly trashy. They all printed fine btw.

Here are the results:
1. https://www.printables.com/de/model/498457-anycubic-kobra-neo-fan-duct
![fan1](https://github.com/1coderookie/KobraGoNeoInsights/assets/17517687/9a88b005-e488-465a-98ce-bd6361edd5bc)
![1 1](https://github.com/1coderookie/KobraGoNeoInsights/assets/17517687/9e0fe626-dd1f-4264-9dd1-38aa0d5f0d19)
![1 2](https://github.com/1coderookie/KobraGoNeoInsights/assets/17517687/61b50649-ef1a-4f03-89b3-35e5e636ff18)

2. https://www.thingiverse.com/thing:5985654
![fan2](https://github.com/1coderookie/KobraGoNeoInsights/assets/17517687/9f316e82-f437-47de-bbbf-55faf409b43a)
![2 1](https://github.com/1coderookie/KobraGoNeoInsights/assets/17517687/992175b3-1d42-4701-977e-b041f7583594)
![2 2](https://github.com/1coderookie/KobraGoNeoInsights/assets/17517687/47bc512c-6a73-4194-8d1a-e66c90f9246a)

3. https://www.printables.com/de/model/470004-anycubic-kobra-neo-fan-duct
![fan3](https://github.com/1coderookie/KobraGoNeoInsights/assets/17517687/d0261c48-96c9-4917-aa6d-4f7cce192481)
![3 1](https://github.com/1coderookie/KobraGoNeoInsights/assets/17517687/25be2dde-5577-48c8-abeb-7c9d4716ba74)
![3 2](https://github.com/1coderookie/KobraGoNeoInsights/assets/17517687/4514aeb2-b907-407d-ac3b-6c7ad3952f04)

I think fan number 3 works the best to be honest. 2 is good as well, and I think the worst one is number 1. They are all very similar tho, so that can mean two things:
- Any custom fan is better than the stock fan, so just pick the one that you like the most
- My testing is wrong. If that's the case I'm guessing the fan height is a probable cause here. Lowering all of these by just a few mm will likely help.

**Reply by NuclearPhoenixx** [reply]
Just updated the comment with the results.

**Reply by 1coderookie** [reply]
Thanks, will look into it more at the weekend (hopefully).  
However, I personally think that the stock fan duct really does a bad job and I had better results with any other duct I've tested in the past - which actually only were three: that circular one I'm using right now, then a modified version of [this one](https://www.printables.com/de/model/378928-anycubic-kobra-neo-fan-duct) (which I adapted to fit my special setup with that longer heatbreak and the wires of the parts sticking out more to the side) before that circular one and then that little extension cap you stick on the stock duct you can find on thingiverse (iirc). I used that one in the first place as I wasn't even able to print anything else properly, but it only made it a liiiiiitle bit better. So I printed out one crappy duct then using that cap and then I worked my way up so to say.. ;)
